CMakeLists.txt

```diff
@@ -137,4 +137,10 @@ if (NOT_SUBPROJECT)
       DESTINATION
         ${RAPIDFUZZ_CMAKE_CONFIG_DESTINATION}
     )
-endif(NOT_SUBPROJECT)
\ No newline at end of file
+    
+    set(CPACK_PACKAGE_VENDOR "Max Bachmann")
+    set(CPACK_PACKAGE_CONTACT "kontakt@maxbachmann.de")
+    set(CPACK_GENERATOR "DEB")
```

Comment:
I have now experience with cpack, but should this really be hardcoded? e.g. catch2 does not specify this value and I assume users on platforms like fedora would need to specify the `CPACK_GENERATOR` rpm

Reply:
I have no idea, as I'm primarily on Ubuntu.  I know you can set it to multiple values.  For some of my projects, I have this:
    set(CPACK_GENERATOR "DEB;RPM")
...but then it requires that you also have the RPM tools installed, otherwise `make package` will fail.

Reply:
I just tested this on my fedora machine:
```
cmake .. -DCMAKE_BUILD_TYPE=Release
cpack -G DEB .
cpack -G RPM .
```
successfully creates:
```
rapidfuzz-1.0.0-Linux.deb
rapidfuzz-1.0.0-Linux.rpm
```
I think we should probably just not set `CPACK_GENERATOR`. This way it appears to default to `set(CPACK_GENERATOR "STGZ;TGZ;TZ")` and when packaging for a specific platform the user has to directly call `cpack` with the corresponding platform.